bugbot/user_activity.py

```diff
@@ -157,6 +172,26 @@ def check_users(
 
     def get_status_from_bz_user(self, user: dict) -> UserStatus:
         """Get the user status from a Bugzilla user object."""
+        if user["creation_time"] > self.new_user_limit:
+            if not user["can_login"]:
+                return UserStatus.DISABLED
+
+            if user["creation_time"] > self.new_user_seen_limit:
+                return UserStatus.ACTIVE
+
+            if (
+                user["last_seen_date"] is None
+                or user["last_seen_date"] < self.new_user_seen_limit
+            ):
+                return UserStatus.ABSENT
+
+            if (
+                user["last_activity_time"] is None
+                or user["last_activity_time"] < self.new_user_activity_limit
+            ):
+                return UserStatus.INACTIVE
+
+            return UserStatus.ACTIVE
```

Comment:
You can reuse most of the existing code if you introduce temporary variables to store the limits.
E.g.:
```Python
is_new_user = user["creation_time"] > self.new_user_limit

seen_limit = self.seen_limit if not is_new_user else self.new_user_seen_limit
activity_limit = self.activity_limit if not is_new_user else self.new_user_activity_limit

if user["creation_time"] > seen_limit:
    return UserStatus.ACTIVE

...
```

Reply:
Good point, done here: b797cebe772d46bdf766c2882958987ed21edc7f